best-practices/styling/index.html

```diff
@@ -1016,17 +1054,180 @@ <h3>Wrapping and breaking text</h3>
+⠀⠀⠉⠊⠑⠛⠕⠎⠐⠜⠀⠐⠣⠠⠠⠥⠇⠁⠉⠐⠜⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠺⠺⠺⠲⠥⠇⠁⠉⠙⠊⠛⠊⠞⠁⠇⠲⠕⠗⠛⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠃⠗⠁⠊⠇⠇⠑⠈⠁⠥⠇⠁⠉⠙⠊⠛⠊⠞⠁⠇⠲⠕⠗⠛⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠉⠕⠝⠎⠑⠚⠕⠀⠠⠊⠃⠑⠗⠕⠁⠍⠑⠗⠊⠉⠁⠝⠕⠀⠙⠑⠇⠀⠠⠃⠗⠁⠊⠇⠇⠑⠀
+⠀⠀⠐⠣⠠⠠⠉⠊⠃⠐⠜⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠁⠙⠞⠁⠈⠁⠕⠝⠉⠑⠲⠑⠎⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠼⠁</code></pre>
+				</aside>
+				<div class="issue" data-number="93" title="Need HTML and CSS of avoiding page breaks within
 				an element"><span/></div>
-
-				<div class="issue" data-number="93" title="Seeking an example of avoiding page breaks between
+				<aside class="example" title="Spatially arranged horizontal equations"></aside>
+					<p>In order to obtain the following desired layout for the above HTML code,</p>
+					<pre><code class="formatted-braille">⠀⠀⠀⠀⠀⠠⠙⠕⠥⠛⠇⠁⠎⠀⠠⠁⠙⠁⠍⠎⠒⠀⠠⠁⠀⠠⠏⠊⠕⠝⠑⠑⠗⠀⠕⠋⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠠⠎⠉⠊⠤⠠⠋⠊⠀⠠⠉⠕⠍⠑⠙⠽⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠙⠕⠥⠛⠇⠁⠎⠀⠠⠁⠙⠁⠍⠎⠀⠺⠁⠎⠀⠁⠀⠠⠃⠗⠊⠞⠊⠎⠓⠀⠁⠥⠞⠓⠕⠗⠀⠀⠀
+⠃⠑⠎⠞⠀⠅⠝⠕⠺⠝⠀⠋⠕⠗⠀⠓⠊⠎⠀⠛⠗⠕⠥⠝⠙⠃⠗⠑⠁⠅⠊⠝⠛⠀⠺⠕⠗⠅⠀⠀
+⠠⠞⠓⠑⠀⠠⠓⠊⠞⠉⠓⠓⠊⠅⠑⠗⠄⠎⠀⠠⠛⠥⠊⠙⠑⠀⠞⠕⠀⠞⠓⠑⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠛⠁⠇⠁⠭⠽⠂⠀⠁⠀⠎⠑⠗⠊⠑⠎⠀⠞⠓⠁⠞⠀⠉⠕⠍⠃⠊⠝⠑⠙⠀⠎⠉⠊⠑⠝⠉⠑⠀
+⠋⠊⠉⠞⠊⠕⠝⠀⠺⠊⠞⠓⠀⠙⠗⠽⠀⠠⠃⠗⠊⠞⠊⠎⠓⠀⠓⠥⠍⠕⠗⠲⠀⠠⠃⠕⠗⠝⠀⠀
+⠊⠝⠀⠼⠁⠊⠑⠃⠂⠀⠠⠁⠙⠁⠍⠎⠀⠓⠁⠙⠀⠁⠀⠅⠝⠁⠉⠅⠀⠋⠕⠗⠀⠺⠑⠁⠧⠊⠝⠛
+⠉⠕⠍⠏⠇⠑⠭⠂⠀⠏⠓⠊⠇⠕⠎⠕⠏⠓⠊⠉⠁⠇⠀⠟⠥⠑⠎⠞⠊⠕⠝⠎⠀⠁⠃⠕⠥⠞⠀⠀
+⠇⠊⠋⠑⠀⠁⠝⠙⠀⠑⠭⠊⠎⠞⠑⠝⠉⠑⠀⠊⠝⠞⠕⠀⠓⠊⠎⠀⠎⠞⠕⠗⠊⠑⠎⠀⠊⠝⠀⠁
+⠺⠁⠽⠀⠞⠓⠁⠞⠀⠺⠁⠎⠀⠃⠕⠞⠓⠀⠁⠉⠉⠑⠎⠎⠊⠃⠇⠑⠀⠁⠝⠙⠀⠁⠃⠎⠥⠗⠙⠲
+⠠⠓⠊⠎⠀⠥⠝⠊⠟⠥⠑⠀⠎⠞⠽⠇⠑⠀⠝⠕⠞⠀⠕⠝⠇⠽⠀⠗⠑⠎⠓⠁⠏⠑⠙⠀⠞⠓⠑⠀
+⠇⠁⠝⠙⠎⠉⠁⠏⠑⠀⠕⠋⠀⠎⠉⠊⠑⠝⠉⠑⠀⠋⠊⠉⠞⠊⠕⠝⠀⠃⠥⠞⠀⠁⠇⠎⠕⠀⠀⠀
+⠊⠝⠋⠇⠥⠑⠝⠉⠑⠙⠀⠁⠀⠛⠑⠝⠑⠗⠁⠞⠊⠕⠝⠀⠕⠋⠀⠺⠗⠊⠞⠑⠗⠎⠀⠁⠝⠙⠀⠀
+⠉⠕⠍⠑⠙⠊⠁⠝⠎⠀⠺⠓⠕⠀⠁⠙⠍⠊⠗⠑⠙⠀⠓⠊⠎⠀⠺⠊⠞⠀⠁⠝⠙⠀⠀⠀⠀⠀⠀⠀
+⠉⠗⠑⠁⠞⠊⠧⠊⠞⠽⠲⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶⠶
+⠀⠀⠀⠀⠠⠞⠓⠑⠀⠠⠕⠗⠊⠛⠊⠝⠀⠕⠋⠀⠠⠞⠓⠑⠀⠠⠓⠊⠞⠉⠓⠓⠊⠅⠑⠗⠄⠎⠀⠀
+⠀⠀⠀⠀⠠⠛⠥⠊⠙⠑⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠠⠞⠓⠑⠀⠉⠕⠝⠉⠑⠏⠞⠀⠋⠕⠗⠀⠠⠞⠓⠑⠀⠠⠓⠊⠞⠉⠓⠓⠊⠅⠑⠗⠄⠎⠀⠀⠀
+⠠⠛⠥⠊⠙⠑⠀⠞⠕⠀⠞⠓⠑⠀⠠⠛⠁⠇⠁⠭⠽⠀⠁⠇⠇⠑⠛⠑⠙⠇⠽⠀⠉⠁⠍⠑⠀⠞⠕⠀
+⠠⠁⠙⠁⠍⠎⠀⠙⠥⠗⠊⠝⠛⠀⠁⠀⠙⠗⠥⠝⠅⠑⠝⠀⠝⠊⠛⠓⠞⠀⠊⠝⠀⠀⠀⠀⠀⠀⠀⠀
+⠠⠊⠝⠝⠎⠃⠗⠥⠉⠅⠂⠀⠠⠁⠥⠎⠞⠗⠊⠁⠲⠀⠠⠺⠓⠊⠇⠑⠀⠇⠽⠊⠝⠛⠀⠊⠝⠀⠁⠀
+⠋⠊⠑⠇⠙⠂⠀⠎⠞⠁⠗⠊⠝⠛⠀⠥⠏⠀⠁⠞⠀⠞⠓⠑⠀⠎⠞⠁⠗⠎⠂⠀⠓⠑⠀⠃⠑⠛⠁⠝
+⠏⠕⠝⠙⠑⠗⠊⠝⠛⠀⠞⠓⠑⠀⠥⠝⠊⠧⠑⠗⠎⠑⠀⠁⠝⠙⠀⠺⠓⠁⠞⠀⠅⠊⠝⠙⠀⠕⠋⠀
+⠛⠥⠊⠙⠑⠀⠎⠕⠍⠑⠕⠝⠑⠀⠺⠕⠥⠇⠙⠀⠝⠑⠑⠙⠀⠞⠕⠀⠝⠁⠧⠊⠛⠁⠞⠑⠀⠊⠞⠲
+⠠⠁⠇⠞⠓⠕⠥⠛⠓⠀⠠⠁⠙⠁⠍⠎⠀⠇⠁⠞⠑⠗⠀⠚⠕⠅⠑⠙⠀⠞⠓⠁⠞⠀⠞⠓⠑⠀⠀⠀
+⠙⠑⠞⠁⠊⠇⠎⠀⠕⠋⠀⠞⠓⠁⠞⠀⠝⠊⠛⠓⠞⠀⠺⠑⠗⠑⠀⠓⠁⠵⠽⠂⠀⠊⠞⠀⠇⠑⠙⠀
+⠞⠕⠀⠞⠓⠑⠀⠉⠗⠑⠁⠞⠊⠕⠝⠀⠕⠋⠀⠞⠓⠑⠀⠃⠑⠇⠕⠧⠑⠙⠀⠋⠊⠉⠞⠊⠕⠝⠁⠇
+⠛⠥⠊⠙⠑⠃⠕⠕⠅⠀⠞⠓⠁⠞⠀⠺⠕⠥⠇⠙⠀⠙⠑⠋⠊⠝⠑⠀⠍⠥⠉⠓⠀⠕⠋⠀⠓⠊⠎⠀
+⠉⠁⠗⠑⠑⠗⠲⠀⠠⠞⠓⠊⠎⠀⠍⠕⠍⠑⠝⠞⠀⠕⠋⠀⠎⠑⠗⠑⠝⠙⠊⠏⠊⠞⠽⠀⠊⠎⠀⠀
+⠕⠋⠞⠑⠝⠀⠉⠊⠞⠑⠙⠀⠁⠎⠀⠕⠝⠑⠀⠕⠋⠀⠞⠓⠑⠀⠍⠕⠎⠞⠀⠋⠁⠍⠕⠥⠎⠀⠀⠀
+⠦⠑⠥⠗⠑⠅⠁⠴⠀⠍⠕⠍⠑⠝⠞⠎⠀⠊⠝⠀⠇⠊⠞⠑⠗⠁⠗⠽⠀⠓⠊⠎⠞⠕⠗⠽⠲⠀⠀⠀
+⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛
+⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠀⠀⠠⠁⠙⠁⠍⠎⠄⠀⠁⠃⠊⠇⠊⠞⠽⠀⠞⠕⠀⠃⠇⠑⠝⠙⠀⠞⠓⠑⠀⠍⠥⠝⠙⠁⠝⠑⠀⠀
+⠺⠊⠞⠓⠀⠞⠓⠑⠀⠉⠕⠎⠍⠊⠉⠀⠺⠁⠎⠀⠁⠀⠎⠊⠛⠝⠁⠞⠥⠗⠑⠀⠞⠗⠁⠊⠞⠀⠕⠋
+⠓⠊⠎⠀⠺⠗⠊⠞⠊⠝⠛⠲⠀⠠⠺⠓⠑⠞⠓⠑⠗⠀⠊⠞⠀⠺⠁⠎⠀⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
+⠃⠥⠗⠑⠁⠥⠉⠗⠁⠞⠊⠉⠀⠁⠇⠊⠑⠝⠀⠗⠁⠉⠑⠀⠞⠗⠽⠊⠝⠛⠀⠞⠕⠀⠀⠀⠀⠀⠀⠀
+⠙⠑⠍⠕⠇⠊⠎⠓⠀⠠⠑⠁⠗⠞⠓⠀⠞⠕⠀⠍⠁⠅⠑⠀⠺⠁⠽⠀⠋⠕⠗⠀⠁⠝⠀⠀⠀⠀⠀⠀
+⠊⠝⠞⠑⠗⠎⠞⠑⠇⠇⠁⠗⠀⠓⠊⠛⠓⠺⠁⠽⠀⠕⠗⠀⠁⠀⠙⠑⠏⠗⠑⠎⠎⠑⠙⠀⠀⠀⠀⠀</code></pre>
+				</aside>				
+				<div class="issue" data-number="93" title="Need HTML and CSS of avoiding page breaks between
 				two elements"><span/></div>
-
-				<div class="issue" data-number="93" title="Seeking an example of avoiding orphan or widow
+				<aside class="example" title="THE END kept on last line page so it stays with other material (rather than with a blank before)"></aside>
```

Comment:
I'm not sure if we're going to be able to fulfill this requirement with CSS. It can not be done with `page-break-after`.

Reply:
@bertfrees Would `page-break-before: avoid` not satisfy the requirement of this kind of formatting?

Reply:
I don't think so. The margin requirement would have to be violated. I'm going to look for a way to do it, but I don't know if it's possible.

Reply:
I still haven't found a solution yet. The [fragmentation module](https://www.w3.org/TR/css-break-3) doesn't seem to account for this use case. For regular (screen) CSS, one might be able to achieve something with [flexbox](https://www.w3.org/TR/css-flexbox), but for braille, there is the additional complexity that margins need to be multiples of braille cells, so I don't know if I should even attempt to explore this path.

Reply:
Going to comment out the example for now.